capstone/src/main/java/com/google/sps/Objects/Challenge.java

```diff
@@ -10,13 +10,15 @@
   private final long dueDate;
   private final Badge badge;
   private final ArrayList<String> usersCompleted;
+  private final long id;
 
-  public Challenge(String challengeName, long dueDate, Badge badge,
-      ArrayList<String> usersCompleted) {
+  public Challenge(
+      String challengeName, long dueDate, Badge badge, ArrayList<String> usersCompleted, long id) {
```

Comment:
Could we just store the badgeId since the badge isn't being shown with the challenge?

Reply:
Aren't we showing the badge icon with the weekly challenge?